### ISSUE TITLE
NumLock bug with screen reader

### ISSUE THREAD

**Issue by WyzrdX** [pop-os/cosmic-epoch#2128]
**Describe the bug**
I have the numlock set to on at boot, yet when I go to use the keypad, I get no response from the keys. Turning numlock off and back on has no effect. Log out, restart, and lock has no effect.

I did figure out the cause. When Screen reader is active, keypad does not work at all. But turn off screen reader and keypad works fine.

**To Reproduce**
1. Turn on screen reader.
2. Turn on NumLock,
3. Attempt to type numbers on keypad. (No response every time).
4. No error message at all just unresponsive.
5. Turn screen reader off and repeat 2 & 3 and issue is resolved except when typing in a .txt file (so far)

**Expected behavior**
Turning on NumLock allows number pad to be used while screen reader is active.

**Screenshots**
No screenshots

**Desktop (please complete the following information):**
 - OS: Pop OS 24.04 Cosmic Alpha 7
